### PR TITLE
Aligning create buttons and cards

### PR DESCRIPTION
**Pull request by Langster2323**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -47,6 +47,18 @@
    }
  }
 
+ // .card-photo-alignment {
+ //   display: flex;
```
Comment (kate-walters) [inline app/assets/stylesheets/application.scss]: Make sure to take out any comments if you're not planning on using them in this PR

**Comment by kate-walters** [comment]
👍

**Comment by rluck0419** [comment]
take a look at the remaining houndci-bot notes - once those are good, it should be good to merge 👍

File: app/views/dashboard/index.html.erb [hunk 2]
```diff
@@ -1,32 +1,38 @@
...
-  <%= link_to "Create Note", new_note_path, class: "btn right" %>
-</div>
-<div class="row">
-  <%= link_to "Create Photo", new_photo_path, class: "btn right col 4" %>
+  <div class="col l6 m6">
+    <%= link_to "Create a Note", new_note_path, class: "btn" %>
+  </div>
+  <div class="col l6 m6">
+    <%= link_to "Create a Photo", new_photo_path, class: "btn" %>
+  </div>
```
Comment (rluck0419) [inline app/views/dashboard/index.html.erb]: create note & create photo

File: features/category_creation.feature [hunk 3]
```diff
@@ -6,7 +6,7 @@ Background: An Admin Logs In and Goes to Category Index
     And I fill in "Email" with "admin@example.com"
     And I fill in "Password" with "password"
     And I press "Submit"
-    Then I should see "Create Note"
+    Then I should see "Create a Note"
```
Comment (rluck0419) [inline features/category_creation.feature]: create note

File: features/category_traversal.feature [hunk 4]
```diff
@@ -6,7 +6,7 @@ Background: An Admin Logs In
     And I fill in "Email" with "admin@example.com"
     And I fill in "Password" with "password"
     And I press "Submit"
-    Then I should see "Create Note"
+    Then I should see "Create a Note"
```
Comment (rluck0419) [inline features/category_traversal.feature]: create note

File: features/dashboard.feature [hunk 5]
```diff
@@ -6,9 +6,9 @@ Background: A User logs in
     And I fill in "Email" with "user@example.com"
     And I fill in "Password" with "password"
     And I press "Submit"
-    Then I should see "Create Note"
+    Then I should see "Create a Note"
```
Comment (rluck0419) [inline features/dashboard.feature]: create note

File: features/dashboard.feature [hunk 6]
```diff
@@ -6,9 +6,9 @@ Background: A User logs in
...
 
 Scenario: User sees photos, notes, creates on dashboard
   And I should see "photo"
   And I should see "note"
-  And I should see "Create Photo"
+  And I should see "Create a Photo"
```
Comment (rluck0419) [inline features/dashboard.feature]: create photo

File: features/user_authentication.feature [hunk 7]
```diff
@@ -7,6 +7,6 @@ Feature: As a User, In order to use the app, I would like to be able to login
     And I fill in "Email" with "user@example.com"
     And I fill in "Password" with "password"
     And I press "Submit"
-    Then I should see "Create Note"
+    Then I should see "Create a Note"
```
Comment (rluck0419) [inline features/user_authentication.feature]: create note

**Pull request merged**